python/mdvtools/dbutils/mdv_server_app.py

```
@@ -557,60 +704,252 @@ def logout():
+                if error_response:
+                    print("------maybe_require_user----ERROR in auth0 validation")
+                    return error_response
+            elif auth_method == "sso":
+                user, error_response = validate_sso_user(request)
+                if error_response:
+                    print("------maybe_require_user----ERROR in sso validation")
+                    return error_response
+            else:
+                return jsonify({"error": "Unknown authentication method"}), 400
+
+            return f(user=user, *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)